Source/Core/VideoCommon/HiresTextures.h

```diff
@@ -29,20 +30,25 @@ class HiresTexture
 
   ~HiresTexture();
 
+  HostTextureFormat GetFormat() const { return m_levels.at(0).format; }
```

Comment:
Please put the implementation in the cpp file with the rest of the functions. Even if it's a simple one liner.